Docker/walletd_config.ini

```diff
@@ -0,0 +1,49 @@
+# File to read Genesis State from
+# genesis-json =
+genesis-json = "/opt/eos/bin/walletd-dir/genesis.json"
+
+# the location of the block log (absolute path or relative to application data dir)
+block-log-dir = "blocks"
+
+# Pairs of [BLOCK_NUM,BLOCK_ID] that should be enforced as checkpoints.
+# checkpoint =
+
+# open the database in read only mode
+readonly = 0
+
+# the location of the chain shared memory files (absolute path or relative to application data dir)
+shared-file-dir = "blockchain"
+
+# Minimum size MB of database shared memory file
+shared-file-size = 8192
+
+# The local IP and port to listen for incoming http connections.
+http-server-endpoint = 0.0.0.0:8888
+
+# The local IP address and port to listen for incoming connections.
+listen-endpoint = 0.0.0.0:9876
+
+# The IP address and port of a remote peer to sync with.
+# remote-endpoint =
+
+# The public IP address and port that should be advertized to peers.
+public-endpoint = 0.0.0.0:9876
+
+# Enable block production, even if the chain is stale.
+enable-stale-production = true
+
+# Percent of producers (0-99) that must be participating in order to produce blocks
+required-participation = false
+
+# ID of producer controlled by this node (e.g. "inita", quotes are required, may specify multiple times)
+# producer-name =
+
+# Tuple of [PublicKey, WIF private key] (may specify multiple times)
+private-key = ["EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV","5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3"]
+
+# Plugin(s) to enable, may be specified multiple times
+# plugin =
+
+plugin = eos::http_plugin
+plugin = eos::wallet_plugin
```

Comment:
Lines 20-21, 44-48 are all that is needed.
Change line 21 to 0.0.0.0:8899

Reply:
Done

Reply:
@heifner I think we can reuse config.ini without having to add a new walletd_config.ini, adding 'walletd-http-server-endpoint = 0.0.0.0:8899' and 'plugin = eos::wallet_plugin' to config.ini

Reply:
@heifner I think we'll modify the C++ module architecture to support reuse of config.ini